molo/yourtips/models.py

```diff
@@ -0,0 +1,180 @@
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+from django.dispatch import receiver
+
+from wagtail.wagtailcore.models import Page
+from wagtail.wagtailcore.fields import StreamField
+from wagtail.wagtailcore import blocks
+from wagtail.wagtailimages.blocks import ImageChooserBlock
+from wagtail.wagtailadmin.edit_handlers import (
+    FieldPanel, StreamFieldPanel, FieldRowPanel,
+    MultiFieldPanel)
+from wagtail.wagtailimages.edit_handlers import ImageChooserPanel
+
+from molo.core.blocks import MarkDownBlock
+from molo.core.utils import generate_slug
+from molo.core.models import (
```

Comment:
Not sure what is best practice here, but some of these molo models imports could be on the same line? Just nit picking though